art/api/models.py

```diff
@@ -1,3 +1,32 @@
 from django.db import models
 
-# Create your models here.
+
+class Asset(models.Model):
+    image = models.ImageField()
```

Comment:
@Awinja-Andela @Mnickii what is the image field for ?

Reply:
@HawiCaesar what image field ? refresh!